alcs-frontend/src/app/features/home/gis/gis-subtasks.component.ts

```diff
@@ -33,9 +31,8 @@ export class GisSubtasksComponent implements OnInit {
       this.statuses = statuses;
     });
 
-    this.$users = this.userService.$users;
     this.userService.$users.subscribe((users) => {
-      this.users = users;
+      this.gisUsers = users.filter((user) => user.clientRoles.includes('GIS'));
```

Comment:
Think this is already the case, but why are we not calling these userRoles or just roles? Client has a different connotation with OAuth2

Reply:
They were called clientRoles on the token, and currently we have 1:1 mapped the token fields to our fields.